config/500-controller.yaml

```diff
@@ -13,48 +13,52 @@
 # limitations under the License.
 
 apiVersion: apps/v1
-kind: StatefulSet
+kind: Deployment
 metadata:
-  name: sample-source-controller-manager
+  name: sample-source-controller
   namespace: knative-samples
   labels:
     samples.knative.dev/release: devel
-    control-plane: sample-source-controller-manager
 spec:
+  replicas: 1
   selector:
-    matchLabels: &labels
-      control-plane: sample-source-controller-manager
-  serviceName: sample-source-controller-manager
+    matchLabels:
+      app: sample-source-controller
   template:
     metadata:
-      labels: *labels
+      labels:
+        app: sample-source-controller
+        samples.knative.dev/release: devel
     spec:
       serviceAccountName: sample-source-controller-manager
```

Comment:
should this be renamed ? Including the SA ?

Reply:
likely, but not in my PR